plugins/SitesManager/templates/_siteWithoutDataTabs.twig

```diff
@@ -12,6 +12,9 @@
             <li class="tab col {{ columnClass }}"><a {% if siteType != constant('Piwik\\Plugins\\SitesManager\\SitesManager::SITE_TYPE_UNKNOWN') and (consentManagerName == false) and (activeTab == '') %} class="active" {% endif %} href="#integrations">{{ 'SitesManager_Integrations'|translate }}</a></li>
             <li class="tab col {{ columnClass }}"><a {% if (siteType == constant('Piwik\\Plugins\\SitesManager\\SitesManager::SITE_TYPE_UNKNOWN') and (activeTab == '')) or consentManagerName %} class="active" {% endif %} href="#tracking-code">{{ 'CoreAdminHome_TrackingCode'|translate }}</a></li>
             <li class="tab col {{ columnClass }}"><a href="#mtm">{{ 'SitesManager_SiteWithoutDataMatomoTagManager'|translate }}</a></li>
+            {% if gtmUsed %}
+                <li class="tab col s2"><a {% if activeTab == 'gtm' %} class="active" {% endif %} href="#google-tag-manager">{{ 'SitesManager_SiteWithoutDataGoogleTagManager'|translate }}</a></li>
```

Comment:
Note: The `s2` class is actually for having 6 columns. So if only GTM or Cloudflare tabs are displayed, there is actually one space free. But as there is no class in materializecss to have 5 columns, we can maybe leave it as is for now...

Reply:
Yes there is nothing which which is available

Reply:
Thanks for highlighting this @sgiehl. I'm sure we're all in favour of refactoring as we go. I'm not sure if this work was pushed through in too much of a hurry to include it, whether we could've approached it better, etc. We can discuss it in an upcoming retro if desired.
Also agreed on the idea that the email should include at least info regarding the tab that the user is looking at. I believe @AltamashShaikh has added copy that refers to the install guides/FAQs related to detected technologies, and while that might not be perfect, it probably is fine for now. We can start getting feedback on this and iterate based on user feedback.
The main reason I'm ok with it for now (rather than working on the email body as part of this ticket) is that I believe this entire email could probably do with a good look at it, and deserves it's own ticket.

Reply:
FYI for e-mail we have #DEV-16664 created internally